server/planning/commands/purge_expired_locks.py

```diff
@@ -8,22 +8,48 @@
 # AUTHORS and LICENSE files distributed with this source code, or
 # at https://www.sourcefabric.org/superdesk/license
 
+import click
 import logging
-from datetime import timedelta
+from datetime import timedelta, datetime
 from eve.utils import date_to_str
```

Comment:
One goal of this project is to remove the eve library. We now have this [date_to_str](https://github.com/superdesk/superdesk-core/blob/async/superdesk/core/utils.py#L69) in our own core library, therefor it would be good to use that import here instead